examples/operator/101_initial_cluster.yaml

```diff
@@ -8,12 +8,12 @@ metadata:
   name: example
 spec:
   images:
-    vtctld: vitess/lite:v8.0.0
-    vtgate: vitess/lite:v8.0.0
-    vttablet: vitess/lite:v8.0.0
-    vtbackup: vitess/lite:v8.0.0
+    vtctld: vitess/lite:v9.0.0
```

Comment:
I see what you did here. Given how close we are to the release I think this is fine.

Reply:
Yea. Otherwise, we'd have to wait to merge the other PR.